### PR TITLE
Add RouterKT

### PR DESCRIPTION
**Pull request by RingoTC**

### CODE REVIEW

**Comment by RingoTC** [comment]
Hi, I trained my model under this framework and you can directly download it from Google Drive for leaderboarding available

https://drive.google.com/drive/folders/1N18XnOE644T3YoWvdBDm6TiZruqp5eMY?usp=sharing

**Comment by ZhijieXiong** [comment]
> Hi, I trained my model under this framework and you can directly download it from Google Drive for leaderboarding available

Hi, thank you for adopting the framework for your model training. I appreciate you sharing your trained model — I’ll add it to the leaderboard as soon as possible.

**Comment by ZhijieXiong** [comment]
Just to confirm, were all five folds of your models trained using the same hyperparameters?

**Comment by ZhijieXiong** [comment]
image

Hi, thanks for sharing this interesting work!
I haven't read the paper in detail yet, but I'm quite shocked by the strong performance of RouterKT in the experiments — the gains over other baselines are really impressive.
Would it be possible to share the training bash scripts or configs?
Also, are there any ablation results available to understand the contribution of each component?

**Comment by RingoTC** [comment]
For sure, you can refer my forked version https://github.com/RingoTC/pyedmine/blob/main/run.sh and the paper https://arxiv.org/abs/2504.08989.
I'm also supervised by this results.

Actually, RouterKT only add a linear router and balance loss based on AKT(also works for other backbones like SimpleKT and CL4KT). Moreover I adopt the default config. If you are interested with our work, you can contact me with ringotc@outlook.com or wechat.
